src/xpcc/driver/temperature/ltc2984_impl.hpp

```diff
@@ -20,6 +20,21 @@ xpcc::Ltc2984<SpiMaster, Cs>::Ltc2984()
 }
 
 template < class SpiMaster, class Cs >
+xpcc::ResumableResult<bool>
+xpcc::Ltc2984<SpiMaster, Cs>::ping()
+{
+	RF_BEGIN();
+	// Store first byte of Custom Sensor Table Data into buffer[0]
+	RF_CALL(readByte(Register::CustomDataTable, buffer[0]));
+	buffer[1] = 0x42;
+	RF_CALL(writeData(Register::CustomDataTable, &buffer[1], 1));
+	RF_CALL(readByte(Register::CustomDataTable, buffer[2]));
+	// Restore first byte of Custom Sensor Table Data from buffer[0]
+	RF_CALL(writeData(Register::CustomDataTable, &buffer[0], 1));
+	RF_END_RETURN(buffer[1] == buffer[2]);
+}
```

Comment:
Colour me skeptical. You sure this works? Doesn‘t `writeData()` overwrite `buffet[0-2]` with `Write` and register address?